lib/server.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+const Hapi = require('hapi');
+const CORE_PLUGINS = [
+    'inert',
+    'vision',
+    '../plugins/status',
+    '../plugins/swagger',
+    '../plugins/logging',
+    '../plugins/stats'
+];
+const CUSTOM_PLUGINS = [
+    'auth',
+    'builds'
+];
+
+/**
+ * Configures & starts up a HapiJS server
+ * @method
+ * @param  {Object}      config
+ * @param  {Object}      config.httpd
```

Comment:
looks like this comment is missing a lot of other stuff like `config.cache`, `auth`, etc.?